package-lock.json

```diff
@@ -0,0 +1,10982 @@
+{
```

Comment:
There shouldn't be a packageLock in the root of our project. Double check this and delete if possible.